napari/layers/image/_image_view.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional
+
+from ...types import ArrayLike, ImageConverter
+
+
+class ImageView:
+    """A raw image and a viewable version of it.
+
+    A very simple class that groups together two related images, the raw one and
+    the viewable one. Its primary purpose right now is just making sure the
+    viewable image is updated when the raw one is changed. And just to group
+    them together and provide convenient access.
+
+    Parameters
+    ----------
+    view_image : ArrayLike
+        Default viewable image, raw is set to the same thing.
+    image_converter : Optional[ImageConverter]
+        If given this is used to convert images from raw to viewable.
+
+    Attributes
+    ----------
+    _raw : ArrayLike
+        The raw image.
+
+    _view : ArrayLike
+        The viewable image, dervied from raw.
+
+    Example:
+    --------
+        # Set the viewable image and raw image to the same thing.
+        image = ImageView(view_image)
+        #  or
+        image.view = view_image
+
+        # Set the raw image, it will compute the new viewable one.
+        image.raw = raw_image
```

Comment:
Same here: use doctest format:

https://numpy.org/doc/stable/docs/howto_document.html

And the section name should be

```
Examples
--------
```

Reply:
Updated to doctest format.